src/pages/workspaces/workspace/Dashboard.js

```diff
@@ -307,7 +307,7 @@ const WorkspaceDashboard = _.flow(
         ]),
         (busy || !tagsList) && spinner({ size: '1rem', style: { marginLeft: '0.5rem' } })
       ]),
-      Utils.canWrite(accessLevel) && div({ style: { marginBottom: '0.5rem' } }, [
+      Utils.canWrite(accessLevel) && !Utils.editWorkspaceError(workspace) && div({ style: { marginBottom: '0.5rem' } }, [
```

Comment:
```suggestion
      !Utils.editWorkspaceError(workspace) && div({ style: { marginBottom: '0.5rem' } }, [
```

`canWrite` is also checked by `editWorkspaceError`.